src/renderer/lib/state.js

```diff
@@ -66,11 +66,10 @@ function getDefaultState () {
 
 function load (cb) {
   appConfig.read(function (err, saved) {
-    if (err || !saved.version) {
+    if (err) {
       console.log('Missing config file: Creating new one')
       onSavedState(null, {
         locale: null,
-        version: config.APP_VERSION,
```

Comment:
This information is completely irrelevant. We always know the version from `package.json`.